### PR TITLE
Support terraform 0.14

### PR DESCRIPTION
**Pull request by dmattia**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -147,7 +147,9 @@ resource "aws_s3_bucket" "origin" {
 }
 
 module "logs" {
-  source                   = "git::https://github.com/cloudposse/terraform-aws-s3-log-storage.git?ref=tags/0.14.0"
+  # TODO: merge https://github.com/cloudposse/terraform-aws-s3-log-storage/pull/42 before submitting, update tag
```
Comment (dmattia) [inline main.tf:150]: This is a dependent PR that should be merged before this one

Comment (hawkesnc) [reply main.tf:150]: It's been merged 👍

File: main.tf [hunk 2]
```diff
@@ -147,7 +147,9 @@ resource "aws_s3_bucket" "origin" {
...
+  # source                   = "git::https://github.com/cloudposse/terraform-aws-s3-log-storage.git?ref=tags/0.14.0"
```
Comment (hawkesnc) [inline main.tf:151]:
```suggestion
  source                   = "git::https://github.com/cloudposse/terraform-aws-s3-log-storage.git?ref=tags/0.15.0"
```

File: main.tf [hunk 3]
```diff
@@ -147,7 +147,9 @@ resource "aws_s3_bucket" "origin" {
...
+  source                   = "git::https://github.com/dmattia/terraform-aws-s3-log-storage.git?ref=patch-1"
```
Comment (hawkesnc) [inline main.tf:152]:
```suggestion
```

File: main.tf (hunk 1 above)
Comment (hawkesnc) [inline main.tf:150]:
```suggestion
```

**Comment by hawkesnc** [comment]
Any updates on this? Thank you for the work! 🙏 🙇‍♂️

**Comment by dmattia** [comment]
Should be ready to go now 😄  @hawkesnc

**Pull request closed without merging**